rfc-177-edition-all-content.md

```diff
@@ -0,0 +1,148 @@
+---
+status: proposed
+implementation: proposed
+status_last_reviewed: 2024-09-09
+---
+
+# Make all content types editionable
+
+## Summary
+
+GOV.UK has a number of types of content published using Whitehall Publisher. Some of these are editionable (i.e. publishers can create a draft version of the document before publishing) whereas others are not editionable (i.e. changes become live immediately).
+
+This RFC proposes using the edition workflow for all publishable entities in Whitehall Publisher.
+
+## Problem
+
+Documents published outside the edition workflow suffer from the following user-facing issues:
+
+* no draft preview (whether for the publisher themselves, or using a shareable preview link for fact checking)
+* no [2i functionality](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/reviewing-and-approving-content\#a-2i-checklist)
+* no scheduled publishing
+* high risk of content errors as changes go live immediately
+* no audit trail
+* validations are not consistent with other document types, e.g. the length of a document summary
```

Comment:
Is there also a problem with inconsistent access control? For example non-editors being able to change organisation social media links?

Reply:
Yes, there are inconsistencies.  Especially the hardcoded VIP editor thing we added a few years ago.